specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -4469,20 +4481,16 @@
     },
     "MaintenanceAlertsProperties": {
       "type": "object",
-      "description": "Properties for host pool update maintenance alerts.",
+      "description": "Customer admins can opt to create additional alerts called maintenance alerts which will be sent out prior to the host pool update kick off. These alerts will enable the customer admin to inform their end users that their host pool update will be kicking off in between 1-5 days based on daysBeforeScheduledUpdate.",
```

Comment:
1. "... These alerts will enable the customer admin**s** to inform..."

Reply:
ok